src/procedures/procedures.11tydata.yaml

```diff
@@ -149,6 +149,14 @@ procedureInstallNvme:
 proceduresInstallingHAOS:
   title: Installing Home Assistant Software on Kit
   steps:
+    - title: Installing the Home Assistant OS Installer
+      image: balena.png
+      content: |
+        <ul>
+        <li>This procedure shows how to install the Home Assistant OS Installer for Yellow using Raspberri&nbsp;Pi&nbsp;Imager. If you are using another imager, download the latest Home&nbsp;Assistant&nbsp;Yellow Installer here:
+         <a href="/power-supply/#eject-the-usb-flash-drive" target="_blank">https://github.com/NabuCasa/yellow-buildroot/releases </a></li>
+          <li>Write the image to your USB flash drive and continue with the step <a href="/power-supply/#eject-the-usb-flash-drive" target="_blank">eject the usb flash drive</a>.</li>
+        </ul>
```

Comment:
Making this a separate step will confuse a lot of people not reading the instructions exactly. I'd suggest to add an info point in the "Installing the Raspberry Pi Imager" step, saying something like:


* If your platform doesn't support the Raspberry Pi Imager you can also use another imager (e.g. balenaEdger) and flash the Home Assistant OS Installer for Yellow directly from [GitHub releases](https://github.com/NabuCasa/yellow-buildroot/releases).

Reply:
done. I had something like that at first, then moved it out into a separate step :-).

Reply:
Can you also drop `public/images/balena.png`, it is no longer required.